### PR TITLE
Gracefully skip tests requiring AES 256 CBC if it's not installed.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: src/main/scala/cromwell/util/Encryption.scala [hunk 1]
```diff
@@ -23,6 +23,22 @@ case object Aes256Cbc {
...
+      ranGen.nextBytes(key)
+      ranGen.nextBytes(iv)
+      init(Cipher.ENCRYPT_MODE, key, iv)
+      init(Cipher.DECRYPT_MODE, key, iv)
+      true
+    } catch {
+      case e: InvalidKeyException =>
+        false
+    }
+  }
```
Comment (mcovarr) [inline src/main/scala/cromwell/util/Encryption.scala]: Should we support a fail fast for missing AES 256 in production code?  That could basically just call this method and bomb out if it returns false.

Comment (kshakir) [inline src/main/scala/cromwell/util/Encryption.scala]: I didn't like my first proof of concept, so I just refactored the check into a pure test utility.

Regarding production code, I'm fine with whatever, but at the moment I'm aiming for a working build to unblock my two PRs. :smile:

There are other devops issues already. For example: I have no idea how one installs JCE unlimited using continuous integration or automation. But, what I don't know could fill a warehouse. I wasn't aware of how to even script installation of Oracle Java until I saw someone else do that too.

**Comment by scottfrazer** [comment]
:+1: thanks for doing this!

**Comment by scottfrazer** [comment]
second reviewer: @Horneth

**Comment by geoffjentry** [comment]
That second commit is baller. :+1:  from me

**Comment by Horneth** [comment]
I like the assume stuff it's cool :+1:

**Comment by mcovarr** [comment]
:+1: ready for squash and merge!

**Comment by kshakir** [comment]
Merging once tests re-pass.

**Pull request merged**